openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -139,7 +139,6 @@ <h5 class="submission__answer__part__text__title">{% trans "The prompt for this
                                   </strong>
                                 {% endif %}
                             </div>
-                            </div>
```

Comment:
how

Reply:
no idea.  I think because we touched neighboring divs.